Update to 5 in STEP and README.md (#5)
MadhavGaur14/skills-review-pull-requests: README.md

Author: changes done

Author: Done

</header>

<!--

>   <<< Author notes: Step 4 >>>

Author: ok

>   Start this step by acknowledging the previous step.

Author: Ok

Author: ```suggestion
  Start this by acknowledging the previous step.
```

  Define terms and link to docs.github.com.
-->

> 

Author: Ok

Author: Okk done

Author: ```suggestion
Yup
```

Author: ok bye

Author: ok

> ## Step 5: Apply suggested changes

Author: ok

Author: ```suggestion
## Step 5: do suggested changes
```

Author: ```suggestion
## Step 5: Apply suggested changes and do it
```

Author: ok

> ## Step 5: Apply suggested changes

Author: Ok

Author: ```suggestion
## Step 5- Apply suggested change
```

Author: ```suggestion
## Step 5: Apply suggested
```

> _Nicely done suggesting changes! :partying_face:_

Author: okk

Author: ```suggestion
_truly done suggesting changes! :partying_face:_
```

Author: > 

yes done

> _Nicely done suggesting changes! :partying_face:_

Author: plz

Author: ```suggestion
_Nicely done suggesting changes.
```

> Now let's see how easy it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

Author: ok

Author: ```suggestion
ok so let's see how easy it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).
```

Author: ```suggestion
Now let's see how hard it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).
```

> Now let's see how easy it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

Author: ok

Author: ```suggestion
Now let's see how hard it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).
```

_Nicely done suggesting changes! :partying_face:_

Now that you have explored the different ways you can review a pull request it is time to learn how to use _suggest changes_.
Now let's see how easy it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

> **What is _suggest changes_?**: This feature enables you to recommend a change to a pull request that the author can commit with the push of a button.

Author: change here

> ### :keyboard: Activity: Apply suggested changes

Author: Ok

Author: ```suggestion:keyboard: Activity: Apply suggested changes
```

Author: ```suggestion
### :keyboard: Activity: ly suggested changes
```

### :keyboard: Activity: Apply suggested changes

> ### :keyboard: Activity: Suggest changes

Author: okk
